scan/lib/scan/options.rb

```diff
@@ -151,6 +151,11 @@ def self.available_options
                                      env_name: "SCAN_FORMATTER",
                                      description: "A custom xcpretty formatter to use",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_format,
```

Comment:
The PR description mentions that the `xcpretty_args` option was added but this is called `xcpretty_format`. Can we actually change this to `xcpretty_args`?

And then probably change other things to match this? 😊